### DISCUSSION TITLE
Lookup device names via local DNS server

### DISCUSSION THREAD

**Discussion by slydogsz** [netalertx/NetAlertX#185] [General]
I am just starting to use Pi.Alert so maybe I am missing where to configure this.
Can it do a dns reverse lookup from a local DNS server and is there someplace where we can set that dns server in the application?

**Comment by jokob-sk** [comment]
I'm not the original author and my network knowledge is limited, but as I understand that's why you can integrate PiAlert with PiHole which then is repsonsible for the DNS lookup. Check the PiHole settings for details.
